easybuild/easyconfigs/p/Perl-bundle-CPAN/Perl-bundle-CPAN-5.36.1-GCCcore-12.3.0.eb

```diff
@@ -2056,22 +2071,6 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/W/WS/WSNYDER'],
         'checksums': ['3973ebdc44682c9c15c776f66e8be242cb4ff1dd52caf43ff446b74d4dccca06'],
     }),
-    ('Sys::Info::Driver::Linux::Device::CPU', '0.7905', {
```

Comment:
@lexming This is being removed here because it's also included later again, but it really must be here, before `Sys::Info`, or the installation of that extension will break.